src/util/toast.ts

```diff
@@ -0,0 +1,11 @@
+import { toast as toastifyToast } from 'react-toastify';
+
+/**
+ * Displays a toast which cannot be duplicated and
+ * continues ticking, even when the window loses focus.
+ */
+export const toast = (content: string, options: any = {}) => {
```

Comment:
Unhappy with the usage of `any` here. How do I use types from `react-toastify`?

Reply:
The appropriate type is `ToastOptions`, just import it from `react-toast`.

Reply:
Thanks, I adapted accordingly